packages/grid/test/keyboard-navigation.test.js

```diff
@@ -1063,6 +1063,13 @@ describe('keyboard navigation', () => {
 
         flushGrid(grid);
 
+        // Force reflow to workaround a Safari rendering issue
```

Comment:
Needed to workaround a Safari rendering issue in the test case. Seems that the test renders funny on master also (even though the oddly passes):

https://user-images.githubusercontent.com/1222264/160112843-27104bdf-d623-4c2d-96a0-cff345fd8f52.mp4